app/views/samples/_form.haml

```diff
@@ -37,7 +37,10 @@
           = text_field_tag :specimen_role, Sample.specimen_role_description(f.object.specimen_role), { :class => 'input-x-large', readonly: true }
 
       = f.form_field :virus_lineage do
-        = f.text_field :virus_lineage, class: 'input-x-large', readonly: !@can_update
+        - if blinded_attribute?(@sample_form, :virus_lineage)
+          .value= blinded_value
+        - else
```

Comment:
This pattern is used three times already. Perhaps this would be a good use case for a blind method that yields when the attribute is not blinded?
```rb
  def blinded_attribute(sample, attr_name, value: nil)
    if blinded_attribute?(sample, attr_name)
      value || blinded_value
    else
      value = sample.__send__(attr_name)
      if block_given?
        yield value
      else
        value
      end
    end
  end
```
It's just an idea and I haven't completely thought it out. Would also need revisiting for the presenter option.